leakcanary-analyzer/build.gradle

```diff
@@ -14,10 +14,10 @@ repositories {
 }
 
 dependencies {
-  compile 'com.squareup.haha:haha:2.0.3'
-  compile project(':leakcanary-watcher')
-  testCompile 'junit:junit:4.12'
-  testCompile 'org.assertj:assertj-core:3.8.0'
+  implementation 'com.squareup.haha:haha:2.0.3'
```

Comment:
Looking at the last two snapshot poms, it seems so:

Compare [leakcanary-analyzer-1.6-20170809.114915-34.pom](https://oss.sonatype.org/content/repositories/snapshots/com/squareup/leakcanary/leakcanary-analyzer/1.6-SNAPSHOT/leakcanary-analyzer-1.6-20170809.114915-34.pom) to [leakcanary-analyzer-1.6-20170809.144341-35.pom](https://oss.sonatype.org/content/repositories/snapshots/com/squareup/leakcanary/leakcanary-analyzer/1.6-SNAPSHOT/leakcanary-analyzer-1.6-20170809.144341-35.pom)